ONSdigital/census-rm-acceptance-tests-2021
# Corrected Wales ICL pack code

# Motivation and Context

The Wales pack code was incorrect. This PR fixes it.

# What has changed



The pack code has been updated in all the places its used

# How to test?



Build images of the action-scheduler and printfilesvc from the branches with the same name as this one. Run tests and check in sftp folder that the print file for wales has the updated name.

https://github.com/ONSdigital/census-rm-action-scheduler/pull/20
https://github.com/ONSdigital/census-rm-print-file-service/pull/3

# Links



https://trello.com/c/DWs4rrCc/925-correct-product-pack-code-for-initial-contact-pack-for-welsh-households-3
# Screenshots (if appropriate):
